src/index.tsx

```diff
@@ -2,8 +2,12 @@ import ReactDOM from 'react-dom';
 import { App } from './App';
 
 import './index.scss';
+import { App2 } from './App2';
 
 ReactDOM.render(
-  <App />,
+  <>
+    <App />
+    <App2 />
+  </>,
```

Comment:
it's nice that you used national colors) yet your code duplicate each other. If you'd like to use two paragraphs with different color you can use it inside one App component, just use two `p`

Reply:
For practice, I implemented App using a Class Component and App2 using a Functional Component.